docker-compose.yml

```diff
@@ -6,10 +6,3 @@ services:
     restart: always
```

Comment:
`unless-stopped` (`always` is really annoying for a local dev env, at least on a Mac)